internal/controller/apiextensions/claim/reconciler.go

```diff
@@ -60,10 +60,11 @@ const (
 
 // Event reasons.
 const (
-	reasonBind      event.Reason = "BindCompositeResource"
-	reasonDelete    event.Reason = "DeleteCompositeResource"
-	reasonConfigure event.Reason = "ConfigureCompositeResource"
-	reasonPropagate event.Reason = "PropagateConnectionSecret"
+	reasonBind               event.Reason = "BindCompositeResource"
+	reasonDelete             event.Reason = "DeleteCompositeResource"
+	reasonCompositeConfigure event.Reason = "ConfigureCompositeResource"
+	reasonClaimConfigure     event.Reason = "COnfigureClaimResource"
```

Comment:
```suggestion
	reasonClaimConfigure     event.Reason = "ConfigureClaim"
```

Reply:
We tend to just call it a "claim", "composite resource claim", or "XRC" rather than a "claim resource".